posthog/hogql/database/schema/query_log_archive.py

```diff
@@ -17,7 +17,7 @@
 QUERY_LOG_ARCHIVE_FIELDS: dict[str, FieldOrTable] = {
     "event_date": DateDatabaseField(name="event_date", nullable=False),
     "event_time": DateTimeDatabaseField(name="event_time", nullable=False),
-    "query_id": StringDatabaseField(name="query_id", nullable=False),
+    "query_id": StringDatabaseField(name="lc_client_query_id", nullable=False),
```

Comment:
do you want this to be `lc_client_query_id`?

Reply:
the follow up is: https://github.com/PostHog/posthog/pull/37579/files